### PR TITLE
Statistic2 0

### PR DESCRIPTION
**Pull request by asqqer**
Lavet statestikpackage

### CODE REVIEW

File: src/main/java/com/outofbounds/empire/Statistics/Controller/OutsideInformations.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import com.outofbounds.empire.Reservation.Models.Reservation;
+import org.springframework.web.bind.annotation.ResponseBody;
+import java.util.List;
+
+public class OutsideInformations {
+
+    public @ResponseBody
+    List<Movie> movies() {
+        return movies();
+    }
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Statistics/Controller/OutsideInformations.java:14]: add spacing between method end and method start

File: src/main/java/com/outofbounds/empire/Statistics/Controller/StatisticsController.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+    @Autowired
+    public ShowingRepository showingRepository;
+
+    @Autowired
+    public MovieRepository movieRepository;
+
+    @Autowired
+    public ShowroomRepository showroomRepository;
+    @CrossOrigin(origins = "http://localhost:8000")
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Statistics/Controller/StatisticsController.java]: add spacing between showrromRepository and Crossorigin annotation

File: src/main/java/com/outofbounds/empire/Statistics/Controller/StatisticsController.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    @RequestMapping(method = RequestMethod.GET, value = "/Statistics")
+    public @ResponseBody
+    List<Statistics> statisticsModels() {
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Statistics/Controller/StatisticsController.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Statistics/Models/Statistics.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
+package com.outofbounds.empire.Statistics.Models;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "statistics")
+public class Statistics {
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Statistics/Models/Statistics.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Statistics/Repositories/StatisticsRepository.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+import com.outofbounds.empire.Statistics.Models.Statistics;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+import java.util.List;
+
+@Transactional
+@Repository
+public interface StatisticsRepository extends CrudRepository<Statistics, Integer> {
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Statistics/Repositories/StatisticsRepository.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Statistics/Repositories/StatisticsRepository.java [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    public List<Statistics> findAll();
+    public Statistics findById(int Id);
+    public Statistics findBymovieID(int movieID);
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Statistics/Repositories/StatisticsRepository.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Statistics/random.java [hunk 7]
```diff
@@ -0,0 +1,4 @@
+package com.outofbounds.empire.Statistics;
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Statistics/random.java]: remove whole class/package

**Review by AnderskdKnudsen — COMMENTED** [review]
Almost weekend!

**Comment by asqqer** [comment]
reviewed

**Review by AnderskdKnudsen — APPROVED** [review]

**Pull request merged**